.github/pull_request_template.md

```diff
@@ -16,3 +16,4 @@ Strike through any lines that are not applicable (`~~line~~`) then check the box
 - [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
 - [ ] Correct, in line with design
 - [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
+- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s)
```

Comment:
Should I also add this to the PR template checklist of the modules ?

Reply:
Maybe reference https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes for more information.

This is not requires for modules, because modules don't have a stable branch.